plugin/restore/mysql/mysql.go

```diff
@@ -124,22 +124,22 @@ func (r *Restore) DeleteOldDatabase(ctx context.Context, database string, suffix
 		return err
 	}
 
-	pitrOldDatabase := getPITROldDatabaseName(database, suffixTs)
+	pitrOldDatabase := GetPITROldDatabaseName(database, suffixTs)
 	if _, err := db.ExecContext(ctx, fmt.Sprintf("DROP DATABASE `%s`;", pitrOldDatabase)); err != nil {
 		return err
 	}
 
 	return nil
 }
 
-// getPITRDatabaseName composes a pitr database name
-func getPITRDatabaseName(database string, suffixTs int64) string {
+// GetPITRDatabaseName composes a pitr database name
+func GetPITRDatabaseName(database string, suffixTs int64) string {
 	suffix := fmt.Sprintf("pitr_%d", suffixTs)
 	return getSafeName(database, suffix)
 }
 
-// getPITROldDatabaseName composes a pitr database name
-func getPITROldDatabaseName(database string, suffixTs int64) string {
+// GetPITROldDatabaseName composes a pitr database name
```

Comment:
these 2 methods have the same comment. It makes reader wonder what's the difference.